terraform/environments/core-network-services/route53.tf

```diff
@@ -3,14 +3,15 @@ locals {
   modernisation-platform-internal-domain = "modernisation-platform.internal"
 
   application-zones = {
-    equip        = "equip.service.justice.gov.uk",
-    ccms-ebs     = "ccms-ebs.service.justice.gov.uk",
-    dacp         = "divorce-section-search.service.justice.gov.uk",
-    mlra         = "maat-libra-administration-tool.service.justice.gov.uk",
-    mojfin       = "laa-finance-data.service.justice.gov.uk",
-    tipstaff     = "tipstaff.service.justice.gov.uk",
-    ncas         = "neutral-citation-allocation.service.justice.gov.uk",
-    pra-register = "parental-responsibility-agreement.service.justice.gov.uk"
+    equip         = "equip.service.justice.gov.uk",
+    ccms-ebs      = "ccms-ebs.service.justice.gov.uk",
+    dacp          = "divorce-section-search.service.justice.gov.uk",
+    mlra          = "maat-libra-administration-tool.service.justice.gov.uk",
+    mojfin        = "laa-finance-data.service.justice.gov.uk",
+    tipstaff      = "tipstaff.service.justice.gov.uk",
+    ncas          = "neutral-citation-allocation.service.justice.gov.uk",
+    pra-register  = "parental-responsibility-agreement.service.justice.gov.uk",
+    delius-jitbit = "jitbit.cr.probation.service.justice.gov.uk"
```

Comment:
We're using this hosted zone to maintain the same URL as the legacy environment. We will delegate DNS to this hosted zone and add a helpdesk.jitbit... record here